lib/impl/AzurePresignTvm.js

```diff
@@ -45,7 +46,7 @@ class AzurePresignTvm extends Tvm {
     const expiryTime = new Date()
     expiryTime.setSeconds(expiryTime.getSeconds() + params.expiryInSeconds)
 
-    const permissions = azure.BlobSASPermissions.parse('r')
+    const permissions = azure.BlobSASPermissions.parse(params.permissions)
```

Comment:
if params.permissions is invalid => we would get a 500, maybe we can have the validation schema take care of this (with a regex or so)